cmake/ExternalProjects.cmake

```diff
@@ -107,6 +116,33 @@ FetchContent_MakeAvailable(zstd_ext)
 target_include_directories(libzstd_static INTERFACE $<BUILD_INTERFACE:${zstd_ext_SOURCE_DIR}/lib>)
 add_library(zstd::libzstd_static ALIAS libzstd_static)
 
+# ZeroMQ
+set(ZEROMQ_LIBRARY ${CMAKE_INSTALL_PREFIX}/lib/libzmq.so)
+ExternalProject_Add(libzeromq_ext
+    GIT_REPOSITORY "https://github.com/zeromq/libzmq.git"
+    GIT_TAG "v4.3.4"
+    CMAKE_CACHE_ARGS "-DCMAKE_INSTALL_PREFIX:STRING=${CMAKE_INSTALL_PREFIX}"
```

Comment:
`libzmq` defaults to [Release](https://github.com/zeromq/libzmq/blob/78ea4ee7878779fb72d3551fb0d79b7b0a5ecd22/CMakeLists.txt#L841) build.